src/ContextMenu.tsx

```diff
@@ -132,14 +140,46 @@ export const ContextMenuItem: FunctionComponent<IContextMenuItem> = props => {
           data={data}
           preventClose={preventClose}
           disabled={disabled}
-          onClick={onClick}
+          onClick={(e, d, t) => {
+            if (onClick) {
+              return onClick(e, d, t);
+            }
+          }}
         >
-          {title && <Text>{title}</Text>}
-          {divider && <Break thickness={1} />}
+          <Flex alignItems="center">
+            {title ? <Text flex={1}>{title}</Text> : null}
```

Comment:
Do we really need to use Text here?

```suggestion
            {title ? <Box flex={1}>{title}</Box> : null}
```